File: tools/bench/langs/strings.cs
using System;

class Program {
  static long Run(long n) {
    string s = "";
    long i = 0;
    while (i < n) {
      s = s + "x";
      i = i + 1;
    }
    return s.Length;
  }

  static void Main(string[] args) {
    Console.WriteLine(Run(long.Parse(args[0])));
  }
}

File: tools/bench/langs/strings.go
package main

import "fmt"

func run(n int64) int64 {
	var s string = ""
	var i int64 = 0
	for i < n {
		s = s + "x"
		i = i + 1
	}
	return int64(len(s))
}

func main() {
	fmt.Println(run(40000))
}
